nextcloud/datadog_checks/nextcloud/data/conf.yaml.example

```diff
@@ -0,0 +1,12 @@
+init_config:
+
+instances:
+  # The URL where nextcloud accepts HTTP requests. This will be used to
+  # fetch statistics.
+  #
+  - url: http://localhost
```

Comment:
Is there a default port we can add to this example?

Reply:
Actually this should be the full URL to the JSON endpoint, and default port is HTTP port. Or maybe I should add the suffix automatically?

Default URL: http://localhost/ocs/v2.php/apps/serverinfo/api/v1/info?format=json